c/validation/adbc_validation.cc

```diff
@@ -318,9 +318,6 @@ void ConnectionTest::TestMetadataGetInfo() {
         case ADBC_INFO_DRIVER_VERSION:
         case ADBC_INFO_VENDOR_NAME:
         case ADBC_INFO_VENDOR_VERSION:
-          // UTF8
-          ASSERT_EQ(uint8_t(0),
-                    reader.array_view->children[1]->buffer_views[0].data.as_uint8[row]);
```

Comment:
Ah, I think I remember what this is now. It's checking the union type code. We should keep this, then...and probably explain it more clearly.